libs/partners/voyageai/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Voyage AI
```

Comment:
going to update this to LangChain to match other packages, and happy to discuss! Just want to make sure I'm not doing something that will create issues for folks using langchain-x packages

Reply:
Sounds good to me!